src/connection.rs

```diff
@@ -552,6 +552,11 @@ impl<T: Read + Write + ScmSocket> HttpConnection<T> {
         self.parsed_requests.pop_front()
     }
 
+    /// Returns if parsed_requests is empty
```

Comment:
The comment should be: `is not empty?`

Reply:
> Can try to increase BUFFER_SIZE to solve it? In the source code `src/connection.rs`, can see a size limit on the read buf.

no, I don't think increase BUFFER_SIZE can solve it.

Reply:
the comment has been fixed